fe/fe-core/src/main/java/com/starrocks/catalog/ArrayType.java

```diff
@@ -25,8 +25,12 @@ public ArrayType(Type itemType) {
         this.itemType = itemType;
     }
 
-    public ArrayType(Type itemType, boolean fromSubQuery) {
-        this.itemType = itemType;
+    public ArrayType(Type itemType, boolean fromDlaEnableDecimalV3) {
```

Comment:
Could you comment when `fromDlaEnableDecimalV3` is false?
Don't know the reason for this change. I see all the usage is passing true.

Reply:
> Could you comment when `fromDlaEnableDecimalV3` is false?
> Don't know the reason for this change. I see all the usage is passing true.

I set them all be true, and then we will  test them, if any one of them is not ok, we just let that one be false which means we don't support that one and give it an unknown type.